packages/hash/subgraph/src/types/identifier.ts

```diff
@@ -1,8 +1,11 @@
-import { VersionedUri } from "@blockprotocol/type-system-node";
 import { OntologyTypeEditionId } from "@hashintel/hash-graph-client";
 import { validate as validateUuid } from "uuid";
 
-export { VersionedUri } from "@blockprotocol/type-system-node";
+// For strange behavior we haven't found the cause of, we are unable to export
+// directly here, and have to import as alias before reexporting the type
+import { VersionedUri as TVersionedUri } from "@blockprotocol/type-system-web";
+// if we don't, the `api` package is unable to use this library.
+export type VersionedUri = TVersionedUri;
```

Comment:
```suggestion
// For strange behavior we haven't found the cause of, we are unable to export
// directly here, and have to import as alias before re-exporting the type
// if we don't, the `api` package is unable to use this library.
import { VersionedUri as TVersionedUri } from "@blockprotocol/type-system-web";
export type VersionedUri = TVersionedUri;
```

The split comment is pretty strange (unless there was some reason for it)

Reply:
No reason other than prettier insisting on a space between the import and export, will apply your suggestion and add the space :)